tests/samples/misc/opengraph_flat_test.json

```diff
@@ -5,9 +5,14 @@
       "og:image": "https://www.eventeducation.com/images/982336_wedding_dayandouan_th.jpg",
       "og:site_name": "Event Education",
       "og:description": "Event Education provides free courses on event planning and management to event professionals worldwide.",
+      "article:publisher": "http://www.facebook.com/PUBLISHER",
```

Comment:
I wonder what was result of parsing ``<meta property="article:publisher" content="http://www.facebook.com/PUBLISHER" />`` before this PR?

Reply:
Here is the result of parsing before that PR:
```
{
    "@context": {
        "og": "http://ogp.me/ns#"
    },
    "@type": "article",
    "og:description": "Event Education provides free courses on event planning and management to event professionals worldwide.",
    "og:image": "https://www.eventeducation.com/images/982336_wedding_dayandouan_th.jpg",
    "og:site_name": "Event Education",
    "og:title": "Himanshu's Open Graph Protocol",
    "og:url": "https://www.eventeducation.com/test.php"
}
```
And after:
```
{
    "@context": {
        "article": "http://ogp.me/ns/article#",
        "og": "http://ogp.me/ns#"
    },
    "@type": "article",
    "article:author": "http://facebook.com/AUTHOR",
    "article:modified_time": "2016-12-13T12:35:52-07:00",
    "article:published_time": "2012-04-04T19:50:00-07:00",
    "article:publisher": "http://www.facebook.com/PUBLISHER",
    "og:description": "Event Education provides free courses on event planning and management to event professionals worldwide.",
    "og:image": "https://www.eventeducation.com/images/982336_wedding_dayandouan_th.jpg",
    "og:site_name": "Event Education",
    "og:title": "Himanshu's Open Graph Protocol",
    "og:url": "https://www.eventeducation.com/test.php"
}
```
Property `article:publisher` would be in the output for the version before only if there is prefix for articles defined in `<head>` tag, but for that test case it doesn't take place. Furthermore I noticed in our test case that this is quite rare situation.

Reply:
@jakubwasikowski thanks for an example, I didn't realize we were missing fields from unknown schemas here 👍